lab_2_keywords_cooccurrence/main.py

```diff
@@ -55,7 +80,10 @@ def calculate_word_degrees(candidate_keyword_phrases: KeyPhrases,
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not type_check(candidate_keyword_phrases, list) or not type_check(content_words, list):
+        return None
+    return {token: sum(len(phrase) for phrase in candidate_keyword_phrases if token in phrase)
```

Comment:
`for` with `if` in another `for`. Too complicated. You can write not at one line

Reply:
Fixed